tests/functions/test_drop_constant_columns.py

```diff
@@ -0,0 +1,17 @@
+import pandas as pd
+import pytest
+from pandas.testing import assert_frame_equal
+
+
+@pytest.mark.functions
+def test_row_to_names(df_constant_columns):
```

Comment:
As a separate comment, let's also get a docstring for the intent of the test function.

Reply:
Ack